src/Symfony/Component/Form/Util/StringUtil.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Util;
+
+/**
+ * @author Issei Murasawa <issei.m7@gmail.com>
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ */
+class StringUtil
+{
+    /**
+     * This class should not be instantiated.
+     */
+    private function __construct()
```

Comment:
i think the most important should be first (which are the constructor and public methods). since the constructor is private it doesn't need to be first in this case as this is not what is of interest.